packages/layout/src/main.rs

```diff
@@ -0,0 +1,180 @@
+use std::env;
+use std::fmt::Write;
+use std::io::{self, Read};
+
+use serde_json::{json, Value};
+
+fn main() {
+    let args: Vec<String> = env::args().skip(1).collect();
+    let action = &args[0];
+    match action.as_str() {
+        "manifest" => manifest(),
+        "transform" => transform(&args[1], &args[2]),
+        other => {
+            eprintln!("Invalid action {other}")
+        }
+    }
+}
+
+fn manifest() {
+    print!("{}", serde_json::to_string(&json!(
+        {
+        "name": "layout",
+        "version": "0.1",
+        "description": "This package provides primitive layout modules.",
+        "transforms": [
+            {
+                "from": "row",
+                "to": ["html"],
+                "arguments": [
+                    {"name": "separator", "default": ",", "description": "The pattern used to separate items in the input content." },
+                    {"name": "gap", "default": "10", "description": "The gap between items given in pixels." },
```

Comment:
Hmm, CSS conventions are moving from absolute units as pixels to rem. We might want to be able to allow for using `rem` instead of `px`, or allow them to add an extension, like `10px` but use `rem` by default.